futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -415,15 +419,28 @@ class LearnerEnrollmentSerializer(
 ):  # pylint: disable=too-many-ancestors
     """Serializer for learner enrollments"""
     course_id = serializers.SerializerMethodField()
+    nafath_id = SerializerOptionalMethodField(field_tags=['nafath_id', 'csv_export'])
 
     class Meta:
         model = CourseEnrollment
         fields = (
             LearnerBasicDetailsSerializer.Meta.fields +
             CourseScoreAndCertificateSerializer.Meta.fields +
-            ['course_id']
+            ['course_id', 'nafath_id']
         )
 
+    def __init__(self, *args: Any, **kwargs: Any):
+        """Initialize the serializer."""
+        super().__init__(*args, **kwargs)
+        self.is_nafath_site = False
```

Comment:
we could have multiple tenants to render, having the flag in `self` does not serve multi-tenants requests